### DISCUSSION TITLE
Make it so Entitys only stack if they can see eachother

### DISCUSSION THREAD

**Discussion by Tizooo** [OmerBenGera/WildStacker#336] [Denied Feature Requests] [answered]
i often have problems that the entitys are stacking with other entitys outside of my fence and then they are free and can run away this is anoying would be cool if u can make it so they have to see eachother a plugin called rosestacker also has this feature which is very helpful

**Comment by OmerBenGera** [comment, marked as answer]
This is not gonna be added, as it's too laggy and cause more trouble than actually being helpful.
